src/src.pro

```diff
@@ -100,14 +100,16 @@ unix:!macx {
 } else:win32: {
     LIBS += -L"C:\\mudlet5_package" \
         -L"C:\\mingw32\\lib" \
-        -llua51 \
+        -llua \
```

Comment:
Unlike `hunspell` I __know__ it has to be a `lua` 5.1 version, isn't it unwise to take out the version?